src/client/components/Card.jsx

```diff
@@ -2,18 +2,27 @@ import React from 'react';
 
 /*
  * {
+ *       index: 3,
  *       word: 'rat',
  *       team: 'red|blue|assassin|civilian',
- *       revealed: true|false
+ *       revealed: true|false,
+ *
+ *       sendToServer: function(message)
  * }
  */
 export default React.createClass({
+  revealCard: function(index) {
+    this.props.sendToServer({
+      type: 'REVEAL_CARD',
+      index: index
+    });
+  },
   render: function() {
     var classes = "card ";
     if (this.props.revealed === true) {
       classes = classes + this.props.team;
     }
-    return <div className={classes}>
+    return <div className={classes} onClick={this.revealCard.bind(this, this.props.index)}>
```

Comment:
interesting thing is that React.createClass auto binds `this` to all of its functions.

i think react throws a console error when you try to actually bind something, i remember having to do `this.revealCard.bind(null, this.props.index)` or something similar